### ISSUE TITLE
Neural Network Models

### ISSUE THREAD

**Issue by Naxalov** [Naxalov/PyTorch_VGG#5]
- [ ] VGG model
- [ ] loss
- [ ] metric
- [ ] optimizer
- [ ] Training loop
